packages/clay-datepicker/README.md

```diff
@@ -0,0 +1,24 @@
+# clay-datepicker
+
+ClayDatepicker
+
+## Setup
+
+1. Install NodeJS >= v0.12.0 and NPM >= v3.0.0, if you don't have it yet. You
```

Comment:
A question: could the Node requirement be bumped to v6? That's the oldest LTS release. [v0.12 hit End-of-life](https://github.com/nodejs/Release) at the end of 2016.

Reply:
Yes, I do not see any problem in being able to go to v6. We just have to change in all to maintain consistency.